about.html

```diff
@@ -0,0 +1,157 @@
+                        <li class="menu-item get-ticket">
+                            <a href="#">Get Ticket</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+            <nav class="mobile-nav">
+                <a href=""><span class="close-button">&#10006;</span></a> 
+                 <ul class="menu-list-mobile" >
+                  <li ><a class="menu-item" href="./index.html#main">Program</a></li><br>
+                  <li ><a class="menu-item" href="./index.html#feature">Speakers</a></li><br>
+                  <li ><a class="menu-item" href="/">Home</a></li><br>
+                  <li ><a class="menu-item" href="#">Get ticket</a></li>
+                 </ul>
+              </nav>
+        </header>
+        <section class="about-intro">
+        <div class="primary-box">
+            <h1 class="primary-h1">Worldwide Musical <br> Concert by 2023</h1>
+        </div>
+        <p class="secondary-text">
+            MuziQ Global Concert brings together the community of experts, creators
+            and producers who compose the good and quality music we all love
+            to hear. This year, we hope to expand our broadcast by streaming on multiple platforms.
+        </p>
+        <p class="about__contact">For more details, please contact <strong>concert@MuziQ.com</strong></p>
+    </div>
+    </section>
+    <section class="about__giveaway-section">
+        <h2 class="section-header about__giveaway-h3">MuziQ Logo</h2>
+
+        <p class="giveaway-details">
+            The logo of MuziQ 2023 was decided by the invited conference speakers for the said conference.
+            After a casual hangout they held in Paris, France, The decided to use the name of the conference
+            as the logo, with multiple colors clipped into the logo text. The idea is to assert that quality 
+            music isn't based on a specific race or group of people.
+        </p>
+
+        <div class="fodera-box">
+            <h3 class="fodera-img">MuziQ</h3>
+        </div>
+    </section>
+
+    <section class="about__past-conferences-section">
+        <h2 class="section-header about__past-conf-section-header">Past Concerts</h2>
+        <p class="past-conf-p">Take a look at the last two music concerst which took place in Paris, France, and in Las Vegas, USA. </p>
+
+        <div class="event-boxes">
+            <div class="event-box">
+                <div class="event-text">
+                    <img src="./images/event1.png" alt="past-event-1">
+                    <!-- <p>2022 <span>MuziQ Concert 2020 in Paris</span></p> -->
+                </div>
+            </div>
+            <div class="event-box">
+                <div class="event-text">
+                    <img src="./images/event1 2.png" alt="past-event-2">
+                    <!-- <p>2021 <span>MuziQ Concert 2022 in Las Vegas</span></p> -->
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- first, remove commented code out of your code base to look more professional and presentable.
- as required this design should be an image with a text centred over it and a red lightly transparent background

your design
![image](https://user-images.githubusercontent.com/60612329/227522447-0b64f2e5-d1d0-4da1-8da7-283a5d1f837c.png)

Reply:
@alexander16108  Kindly explain in detail what needs to be done here because I don't really get you

Reply:
@alexander16108 done with this change